models/marts/core/dimensions/dim_equipe.sql

```diff
@@ -0,0 +1,137 @@
+                    (upper(descricao) like "TECNICO EM SAUDE BUCAL%")
+                    or (upper(descricao) like "AUXILIAR EM SAUDE BUCAL%")
+                then "auxiliares_tecnico_saude_bucal"
+                when
+                    (upper(descricao) like "CIRURGIAO DENTISTA%")
+                    or (upper(descricao) like "CIRURGIAO-DENTISTA%")
+                    or (upper(descricao) like "CIRURGIAODENTISTA%")
+                then "dentista"
+                else "outros_profissionais"
+            end grupo,
+            dense_rank() over (
+                partition by id_unidade, equipe_sequencial
+                order by equipe_p.data_particao desc
+            ) as ordenacao
+        from {{ ref("raw_cnes_web__equipe_profissionais") }} as equipe_p
+        left join {{ ref("raw_datasus__cbo") }} as cbo on cbo.id_cbo = equipe_p.id_cbo
+        where id_municipio = '330455'
+
+    ),
+profissionais_equipe as (
+select * EXCEPT(ordenacao)
+from
+    profissionais_enriquecido pivot (
+        array_agg(distinct id_profissional_sus ignore nulls)
+        for grupo in (
+            'medicos',
+            'enfermeiros',
+            'auxiliares_tecnicos_enfermagem',
+            'agentes_comunitarios',
+            'auxiliares_tecnico_saude_bucal',
+            'dentista',
+            'outros_profissionais'
+        )
+    )
+where ordenacao = 1
+),
+dim_estabelecimentos as (
+    select distinct id_unidade from {{ ref("dim_estabelecimento") }}
+),
+versao_atual as (
+    select max(data_particao) as versao from {{ ref("raw_cnes_web__equipe") }}
+),
+equipe as (
+    select *
+    from {{ ref("raw_cnes_web__equipe") }}
+    where
+        id_municipio = '330455'
+        and data_particao = (select versao from versao_atual)
+),
+dim_segmento as (
+    select id_segmento, segmento_descricao,
+    from {{ ref("raw_cnes_web__segmento") }}
+    where
+        data_particao = (select versao from versao_atual)
+        and id_municipio = '330455'
+),
+dim_area_segmento as (
+    select area.id_area, area.area_descricao, dim_segmento.segmento_descricao
+    from {{ ref("raw_cnes_web__area") }} as area
+    left join dim_segmento on dim_segmento.id_segmento = area.id_segmento
+    where
+        area.data_particao = (select versao from versao_atual)
+        and area.id_municipio = '330455'
+),
+dim_tipo_equipe as (
+    select id_equipe_tipo, equipe_descricao, id_equipe_grupo
+    from {{ ref("raw_cnes_web__equipe_tipo") }}
+    where data_particao = (select versao from versao_atual)
+),
+contato_equipe as (
+    select ine, telefone
+    from {{ ref("raw_plataforma_smsrio__equipe_contato") }}
+)
+select
+    equipe.equipe_ine as ine,
+    equipe.equipe_nome as nome_referencia,
+    equipe.id_unidade as id_unidade_saude,
+    equipe.id_tipo_equipe as id_equipe_tipo,
+    dim_tipo_equipe.equipe_descricao as equipe_tipo_descricao,
+    equipe.id_area,
+    dim_area_segmento.area_descricao,
+    dim_area_segmento.segmento_descricao,
```

Comment:
Achei bem zoneado essa campo da forma que vêm do CNES. Ele é realmente necessário? Se for, conseguimos padronizá-lo?

Reply:
Parece ser só a área programática. Já obtemos isso a partir do estabelecimento. Pode limar
![image](https://github.com/prefeitura-rio/queries-rj-sms/assets/7233184/2a0940bf-c72d-4f17-b086-3478312b9f74)